frontend/src/models/propertyDefinitionsModel.ts

```diff
@@ -361,7 +350,7 @@ export const propertyDefinitionsModel = kea<propertyDefinitionsModelType>([
             }
 
             const propValues: PropValue[] = await api.get(
-                endpoint || getEndpoint(values.currentTeamId, type, propertyKey, eventNames, newInput),
+                constructValuesEndpoint(endpoint, values.currentTeamId, type, propertyKey, eventNames, newInput),
```

Comment:
slightly different here we now always build the endpoint whereas before we'd accept one if it was present... is that ok?

Reply:
We actually still accept the endpoint if present, that logic has just moved up to https://github.com/PostHog/posthog/pull/23689/files?diff=unified#diff-264ccd2aeb4ad3326204b410669dc1c2305b6339254abefd1dcc35d3eb6e3922R128

Previously if you gave an `endpoint` it wouldn't append the the `value` query param which is the input value when searching. We should always include that when sending the request even if some endpoints ignore it